_posts/2020-01-20-hacks.md

```diff
@@ -0,0 +1,101 @@
+* Intellectual Property : with Blockchain projects creative individuals can have the opportunity to share their content online and be able to prove that they are indeed the creators of said content and hold all intellectual property rights regardless of where it ends up, and everyone else would be aware of this too.
+* Charitable organizations are obviously a good thing, especially when they actually use their funds for the intended purposes, which is typically to help people. Unfortunately, that is not always the case, and there have been situations in which charitable organizations have been labeled as corrupt and have been involved in unethical activities. Blockchain technology can help remedy this.
+* Digital IDs : More than 1 billion people worldwide face identity challenges. Creating digital IDs which would give users a way to control their digital identities.
+* Medical recordkeeping: The good news is the medical sector has already been moving away from paper for recordkeeping purposes for years.
+* However, blockchain offers even more safety and convenience. In addition to storing patient records, the patient, who possesses the key to access these digital records, would be in control of who gains access to that data.
+* Weapons tracking : One of the hot-button topics on any news network at the moment is gun control and/or weapons accountability. Blockchain could create a transparent and unchanging registry network that allows law enforcement and the federal government to track gun or weapon ownership, as well as keep a record of weapons sold privately.
+
+### AI/ML
+
+* A system which can do a check-up for emergencies and give short term advice.
+* Classifying articles or news stories as worth reading or not.
+* An app for taxi drivers- which will help them to identify which locations to concentrate and go around in loops in order to get maximum customers.
+* Agriculture has transformed with the advent of irrigation, but irrigation itself requires a certain level of supervision. Develop Irrigation systems which self regulate according to weather patterns.
+* Depression affects about 20% of adolescents by the time they become adults. Develop Interfaces or chatbots that can help counsel people with issues like depression or anxiety issues and suicidal tendencies.
+* Personal Health Assistant : The AI in your phone will figure out exactly when and how to wake you up given your health, sleep patterns and time constraints. It will also look into your eating habits and suggest you restaurants and dishes for keeping you in track.
+
+### Algorithms
+
+* Can you devise new algorithms to improve the control of vehicular traffic in crowded cities?
+* Bitcoin – last year it was claimed that the computing power required to keep the network running consumes as much energy as was used by 159 of the world’s nations. Come up with an implementation which solves the energy problem with your own algorithm.
```

Comment:
Come up with an implementation which solves the energy problem with your own **Consensus** algorithm.